### ISSUE TITLE
Website - Accessibility - Not Able to Tab to Search Results Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#74]
A keyboard only or screen reader user is not able to tab to what look and like links for the mouse user.

The LINKS seem to be DIVs made to act like LINKS. The links should be created with an A HREF so they users can tab to them with a keyboard and also activate with a keyboard too.

**Comment by jfc3-dol** [comment]
Closing issue for now since I can now tab to the search results links in Chrome. Will need to test in IE and Firefox once the page is able to display.
